assets/css/fw.css

```diff
@@ -295,6 +295,22 @@ main>div {
   margin: 0 .3em 0 0;
 }
 
+details {
+  font: 16px;
+  font-style: bold;
+  width: 620px;
+}
+
+details>summary {
```

Comment:
Should `>` have spaces around it?

Reply:
It doesn't need to, I think, but it can. Would you prefer that?

Reply:
It did have and I don't see a reason to make the code less readable.  So yes.

Reply:
Hmm, no: The atom beautifier reverts it.

Reply:
Oh well, if that's "correct" according to some idea of CSS styling, then fine.